docs/reference/ml/anomaly-detection/apis/get-category.asciidoc

```diff
@@ -85,6 +95,18 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
 The value is increased by 10% to enable matching for similar fields that have
 not been analyzed.
 
+// This doesn't use the shared description because there are
+// categorization-specific aspects to its use in this context
+`partition_field_name`::
+(string) The field used to segment the categorization, if per-partition
```

Comment:
We generally put the conditional bit at the beginning of the sentence, so folks can stop reading if it doesn't apply.  I'd suggest:

> If per-partition categorization is enabled, this value identifies the field used to segment the categorization.